.pre-commit-config.yaml

```diff
@@ -1,4 +1,13 @@
 repos:
+  - repo: "https://github.com/pre-commit/pre-commit-hooks"
+    rev: "v4.4.0"
+    hooks:
+      - id: "check-added-large-files"
+      - id: "check-vcs-permalinks"
+      - id: "no-commit-to-branch"
```

Comment:
The default behavior is to block commits to `main` and `master`.

Reply:
I guess this is fine, but isn't this already provided by GitHub branch protection?

Reply:
True, I turned that on just now. But this would stop someone prior to committing, so maybe there's still some value there?

Reply:
Works for me.